tests/model_fields/test_decimalfield.py

```diff
@@ -91,7 +91,10 @@ def test_lookup_really_big_value(self):
         Really big values can be used in a filter statement.
         """
         # This should not crash.
-        Foo.objects.filter(d__gte=100000000000)
+        list(Foo.objects.filter(d__gte=100000000000))
+
+    def test_lookup_decimal_larger_than_max_digits(self):
+        list(Foo.objects.filter(d__lte=Decimal("123456")))
```

Comment:
Same here, my fear is that is not obvious to (future) readers why this line alone is testing anything.
```suggestion
        self.assertEqual(list(Foo.objects.filter(d__lte=Decimal("123456"))), [])
```